app/Filament/Resources/EmulatorResource/RelationManagers/EmulatorReleasesRelationManager.php

```diff
@@ -57,10 +64,11 @@ public function table(Table $table): Table
 
             ])
             ->headerActions([
-
+                Tables\Actions\CreateAction::make(),
             ])
             ->actions([
-
+                Tables\Actions\EditAction::make(),
```

Comment:
There should only be one version set as Minimum at any give time. I'm not sure if that would be easy to do automatically.

Since there isn't a view for releases (just an edit page), should edits be shown in the emulator log?

Reply:
Fixed in latest so only one value can be set to minimum at a time. If any row is set to minimum, any row which has a truthy minimum will flip to false.

> Since there isn't a view for releases (just an edit page), should edits be shown in the emulator log?

I don't think so. I believe this breaks several assumptions made by the audit log page itself.

I think what the best course of action is would be to create an audit log page for emulator releases resources which is accessible via the actions column in the rows, kind of like some of the other resources such as achievements:

![Screenshot 2024-09-23 at 9 01 29 PM](https://github.com/user-attachments/assets/64fa8695-511c-4af4-824b-3366b42fab19)

This is not currently present in latest though, as I haven't explored what would be involved in building a "detached" audit log page such as this.